.github/CODEOWNERS

```diff
@@ -0,0 +1,126 @@
+# Global owner for everything in this repo
+
+- @rt4914
```

Comment:
This should be 

* @rt4914

Reply:
I have changed to above requirement. Thanks